k8s-deploy/Makefile

```diff
@@ -68,12 +70,12 @@ swag: swag-bin
 package: kubefate-without-swag
 	mkdir -p tmp/kubefate;
 	cp -r bin/kubefate *.yaml examples tmp/kubefate;
-	tar -czvf kubefate-k8s-${RELEASE_VERSION}.tar.gz -C tmp/ kubefate;
+	tar -czvf kubefate-k8s-${VERSION}-${ARCH}.tar.gz -C tmp/ kubefate;
 	rm -r tmp;
 
 release: package docker-save
-	mkdir -p release;
-	mv kubefate-k8s-${RELEASE_VERSION}.tar.gz kubefate-${VERSION}.docker release/;
+	mkdir -p release/${ARCH};
+	mv kubefate-k8s-${VERSION}-${ARCH}.tar.gz kubefate-${VERSION}-${ARCH}.docker release/${ARCH};
```

Comment:
We don't need to make a dir with the arch name, let's put everything into the release folder.